client/src/components/screens/Patient/components/PrescriptionsTab/components/Prescriptions.tsx

```diff
@@ -211,6 +211,7 @@ const Prescriptions : React.FC<Props> = ({ prescriptions, clinicalImpressions })
                   <Row>
                     <Col>
                       <Button
+                        className="bordelessButton"
```

Comment:
Also, there is a typo in the class name, it should be `borderlessButton`.
I know you're just using it but could you convert it to `borderless-button` ?

Reply:
Yeah. I'll rename it `button--borderless` to follow a bit more the bem naming